handlers/registration.js

```diff
@@ -31,23 +34,55 @@ module.exports.create = async (event, ctx, callback) => {
   let registrationStatus = data.registrationStatus;
 
   // Check if the event is full
-  if (registrationStatus === 'registered'){
+  if (registrationStatus === 'registered') {
 
     const eventParams = {
       Key: { id: eventID },
       TableName: 'biztechEvents' + process.env.ENVIRONMENT
     }
-    
-    await docClient.get(eventParams).promise()
-    .then(async(event) => {
-      const counts = await helpers.getEventCounts(eventID)
 
-      if (counts.registeredCount >= event.Item.capac){
-        registrationStatus = 'waitlist'
-      }
-    })
+    await docClient.get(eventParams).promise()
+      .then(async (event) => {
+        const counts = await helpers.getEventCounts(eventID);
+
+        if (counts.registeredCount >= event.Item.capac) {
+          registrationStatus = 'waitlist'
+        }
+        return event.Item.ename;
+      }).then(async (eventName) => {
```

Comment:
nit: new line for the `.then` but its ok

Reply:
big bet